appserver/tests/payara-samples/samples/asadmin/src/test/java/fish/payara/samples/asadmin/SetHazelcastConfigurationTest.java

```diff
@@ -39,15 +39,22 @@
  */
 package fish.payara.samples.asadmin;
 
+import fish.payara.nucleus.hazelcast.HazelcastRuntimeConfiguration;
+import fish.payara.samples.Unstable;
+
 import org.glassfish.embeddable.CommandResult;
 import org.junit.Before;
 import org.junit.Test;
-
-import fish.payara.nucleus.hazelcast.HazelcastRuntimeConfiguration;
+import org.junit.experimental.categories.Category;
 
 /**
  * Verifies the correctness of the {@code SetHazelcastConfiguration} command.
  */
+@Category(Unstable.class)
+// Fails from two reasons:
+// 1) Requires completely new domain. Side effects of other tests break this one.
```

Comment:
The only test I see that requires a clean domain is the dataGridEncryptionWarning test, which surely could also be acheived by making the other test that creates the key clean up after itself properly?

Reply:
Both should clean up after their run. Or better we should have some architecture in tests so instead of testing "random" features in random order we could join them to some scenarios, use cases, categories/groups (?) and each category would start with a new domain. Then we can have a control over it, but imagine we would have thousand of tests in payara samples - which of those test result states interact with any other? The current state of things is not maintainable in larger scope.